en/sql/schema/index_stmt.rst

```diff
@@ -327,12 +327,12 @@ The following is a syntax of rebuilding an index.
 
 ::
 
-    ALTER INDEX index_name ON [schema_name.]table_name REBUILD;
+    ALTER INDEX index_name ON [schema_name.]table_name {{[COMMENT index_comment_string] REBUILD} | {COMMENT index_comment_string} | VISIBLE | INVISIBLE};
```

Comment:
Simplifying the syntax seems to be required. In case of changing visibility, COMMENT can not be specified?

Reply:
If I were to express it in one line, I think the current state is the best.